log4j-tools-parent/.mvn

```diff
@@ -0,0 +1 @@
+../.mvn
```

Comment:
Why do we need these symlinks?

Reply:
I was running Maven from IDEA (Maven view).
The IDE is unable to find `.mvn/jvm.config` in the root project if I launch a sub-module.

Nevertheless I should probably remove these.